app/models/case_study.rb

```diff
@@ -0,0 +1,35 @@
+class CaseStudy < ApplicationRecord
+  include SpotlightSearch
+  include Paginator
+
+	validates :name,  presence: true
+
+  belongs_to :created_by, foreign_key: 'created_by_id', class_name: 'User', optional: true
```

Comment:
why is this optional bro? All case studies will be created on the Admin Panel right?